CoffeeChat/Controller/SchedulingPlacesViewController.swift

```diff
@@ -0,0 +1,289 @@
+        nextButton.isEnabled = enable
+        if enable {
+            nextButton.backgroundColor = .backgroundOrange
+            nextButton.layer.shadowColor = UIColor.black.cgColor
+            nextButton.layer.shadowOffset = CGSize(width: 0.0, height: 2.0)
+            nextButton.layer.shadowOpacity = 0.15
+            nextButton.layer.shadowRadius = 2
+        } else {
+            nextButton.backgroundColor = .inactiveGreen
+            nextButton.layer.shadowColor = .none
+            nextButton.layer.shadowOffset = CGSize(width: 0.0, height: 0.0)
+            nextButton.layer.shadowOpacity = 0
+            nextButton.layer.shadowRadius = 0
+        }
+    }
+
+    @objc private func nextButtonPressed() {
+        // TODO implement
+    }
+
+    @objc private func backButtonPressed() {
+        // TODO implement
+    }
+
+
+}
+
+// MARK: - UICollectionViewDelegate
+extension SchedulingPlacesViewController: UICollectionViewDelegate {
+
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        let section = locationSections[indexPath.section]
+        switch section {
+        case .campus(let locations):
+            selectedCampusLocations.append(locations[indexPath.row])
+        case .ctown(let locations):
+            selectedCtownLocations.append(locations[indexPath.row])
+        }
+        (collectionView.cellForItem(at: indexPath) as? SchedulingCollectionViewCell)?.changeSelection(selected: true)
+        updateNext()
+    }
+
+    func collectionView(_ collectionView: UICollectionView, didDeselectItemAt indexPath: IndexPath) {
+        let section = locationSections[indexPath.section]
+        switch section {
+        case .campus(let locations):
+            selectedCampusLocations.removeAll { $0 == locations[indexPath.row] }
+        case .ctown(let locations):
+            selectedCtownLocations.removeAll { $0 == locations[indexPath.row] }
+        }
+        (collectionView.cellForItem(at: indexPath) as? SchedulingCollectionViewCell)?.changeSelection(selected: false)
+        updateNext()
+    }
+
+}
+
+// MARK: - UICollectionViewDataSource
+extension SchedulingPlacesViewController: UICollectionViewDataSource {
+
+    func numberOfSections(in collectionView: UICollectionView) -> Int {
+        2
+    }
+
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        switch locationSections[section] {
+        case .campus(let locations): return locations.count
+        case .ctown(let locations): return locations.count
+        }
+    }
+
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        switch locationSections[indexPath.section] {
+        case .campus(let locations):
+            guard let cell = locationsCollectionView.dequeueReusableCell(withReuseIdentifier: campusReuseIdentifier, for: indexPath) as?
```

Comment:
you can just use the `collectionView` argument instead of directly referencing `locationsCollectionView`